grass7/raster/r.null.all/r.null.all.py

```diff
@@ -0,0 +1,190 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+############################################################################
+#
+# MODULE:       r.null.all
+# AUTHOR(S):    Vaclav Petras <wenzeslaus gmail com>
+#
+# PURPOSE:      Manage null values for all raster maps in a mapset
+# COPYRIGHT:    (C) 2019 by Vaclav Petras and the GRASS Development Team
+#
+#               This program is free software under the GNU General
+#               Public License (>=v2). Read the file COPYING that
+#               comes with GRASS for details.
+#
+#############################################################################
+
+#%module
+#% description: Manages NULL values of raster maps in a mapset or their subset
```

Comment:
```suggestion
#% description: Manages NULL values of raster maps in a mapset or their subset.
```

Reply:
We should probably fix this through out the code base. In addons, most don't have the period, in core, most have but not all. `--script` gives the period.

    grep -IrnE -m 1 description: path/to/code | grep "[\.]$" | wc

The question is if we want the period there. It is not even a sentence.

If I recall correctly, the current status is that the code processing these just adds (or perhaps also) removes the period as needed.